script.js

```diff
@@ -0,0 +1,80 @@
+//sets isO to false so first click is an X, next click an O, then an X, etc.
+var isO = false;
+var h2 = $("h2");
+
+//function containing the events to play the game
+function playTicTac() {
+  var gameBoard = $("#gameBoard");
+  gameBoard.on("click", clickSquare);
+  $("button").on("click", clearBoard);
+  h2.addClass("xs");
+}
+playTicTac();
+
+//variable to gather all of the squares in an array
+var allSquares = $("#one, #two, #three, #four, #five, #six, #seven, #eight, #nine");
+
+function clickSquare(event) {
+  event.preventDefault();
+  //variables that are arrays of potential winning locations. when all three values in the array 1) have a value and 2) have matching values, then that value is the winner!
+  //with more time, I would like to refactor these variables into an object.
+  var colLeft = $("#one, #four, #seven");
+  var colMid = $("#two, #five, #eight");
+  var colRight = $("#three, #six, #nine");
+  var rowTop = $("#one, #two, #three");
+  var rowMid = $("#four, #five, #six");
+  var rowBot = $("#seven, #eight, #nine");
+  var diagOne = $("#one, #five, #nine");
+  var diagTwo = $("#three, #five, #seven");
+
+  //when square is clicked, enter an X or O.
+  if (isO === false) {
+    //add X into square when clicked
+    event.target.innerHTML = "X";
```

Comment:
This function is a little long, I'm might abstract this code here into another funciton